src/QueryBuilder.php

```diff
@@ -153,22 +153,22 @@ public function __set($name, $value)
         $this->subject->{$name} = $value;
     }
 
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->subject[$offset]);
     }
 
-    public function offsetGet($offset)
+    public function offsetGet($offset): bool
```

Comment:
offsetGet should return `mixed` (Can return all value types) type

https://www.php.net/manual/en/arrayaccess.offsetget.php